Cargo.toml

```diff
@@ -98,12 +98,16 @@ winapi = { version = "0.3.9", features = [
 wio = { version = "0.2.2" }
 skia-safe = { version = "0.68.0", features = ["gl", "d3d", "textlayout"] }
 
-[target.'cfg(not(target_os = "windows"))'.dependencies]
-skia-safe = { version = "0.68.0", features = ["gl", "textlayout"] }
-
 [target.'cfg(target_os = "macos")'.dependencies]
 icrate = { version = "0.0.4", features = [ "apple", "Foundation", "Foundation_NSThread", "AppKit", "AppKit_NSColor", "AppKit_NSEvent", "AppKit_NSView", "AppKit_NSWindow", "AppKit_NSViewController", "AppKit_NSMenu", "AppKit_NSMenuItem", "AppKit_NSOpenPanel", "AppKit_NSScreen", "Foundation_NSArray" ] }
 objc2 = "0.4.1"
+metal = "0.27.0"
+skia-safe = { version = "0.68.0", features = ["metal", "gl", "textlayout"] }
```

Comment:
It think you should be able to use the standard `gl`, `textlayout`, feature here. We don't use anything from the `metal` and `gl` features, even if the rendering is done through that.

There's no `metal`, `gl`, `textlayout` binary combination as far as I can see https://github.com/rust-skia/skia-binaries/releases, so it would slow down the compilation.

Reply:
Though there is no `metal`, `gl`, `textlayout` combination there, this is exactly what we want. Just like on Windows, there are two renderer, OpenGL and D3D. There are also two renderer, OpenGL and Metal on macOS. Without either one, it does not compile.

I don't know why there is no such combination. Maybe due to OpenGL is obsolete on macOS.

Reply:
I was probably wrong, and it's indeed needed. I remembered wrong how it was done on Window.

I guess we have to ask for binaries from `skia-safe` then.